<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Import animated file:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="WLabelElide" name="lblFileName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono, Consolas, Courier New</family>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Import mode:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbImportAsFrameLayer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Frames from animation are imported in a single layer defined as an animated layer</string>
         </property>
         <property name="text">
          <string>As an animated frame la&amp;yer</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hsKeyFrameStep">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Each imported frame is a keyframe: the value define number of frame the keyframe generate</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>48</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbKeyFrameStep">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Each imported frame is a keyframe: the value define number of frame the keyframe generate</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>48</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="rbImportAsLayers">
       <property name="toolTip">
        <string>One layer is created for each animation frame</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>As s&amp;tacked layers</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="rbImportSingleFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>A single frame is opened in a new document</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>A sin&amp;gle frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hsKeyFrameNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbKeyFrameNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="rbImportKritaDefault">
       <property name="toolTip">
        <string>Use default importer plugin from Krita</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Default &amp;Krita's importer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WLabelElide</class>
   <extends>QLabel</extends>
   <header>bulicommander.pktk.widgets.wlabelelide</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dbbxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsKeyFrameStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbKeyFrameStep</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsKeyFrameNumber</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbKeyFrameNumber</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbKeyFrameStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsKeyFrameStep</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>801</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbKeyFrameNumber</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsKeyFrameNumber</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
